substrate/frame/staking/src/lib.rs

```diff
@@ -382,6 +382,34 @@ pub struct ActiveEraInfo {
 	pub start: Option<u64>,
 }
 
+/// Parameters of the unbonding queue mechanism.
+#[derive(PartialEq, Eq, Copy, Clone, Encode, Decode, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+#[scale_info(skip_type_params(T))]
+pub struct UnbondingQueue {
```

Comment:
```suggestion
pub struct UnbondingQueueConfig {
```

otherwise the name is misleading